app/components/dns-record/dns-record-name.tsx

```diff
@@ -0,0 +1,27 @@
+import type { FC } from 'react';
+import React from 'react';
+import { Flex, Text } from '@chakra-ui/react';
+
+interface DnsRecordNameProps {
+  name: string;
+}
+
+const DnsRecordName: FC<DnsRecordNameProps> = (props) => {
+  const { name } = props;
```

Comment:
I don't see how is it cleaner. In my opinion my version is more cleaner, since it show that the props object is passed and then decompose or for certain params

Reply:
Would that be the same if you specify it on the params? When you write it from there it already specifies that it's a prop? I was just wondering why add an extra line for a process you can do directly from the parameter?

Reply:
`FC` has fallen out of favour https://github.com/facebook/create-react-app/pull/8177